umh-core/test/fsm/benthos/benthos_test.go

```diff
@@ -1220,11 +1220,82 @@ var _ = Describe("BenthosInstance FSM", func() {
 			// Now we should get the error
 			Expect(recErr).To(HaveOccurred())
 			Expect(recErr.Error()).To(ContainSubstring(backoff.PermanentFailureError))
-			Expect(reconciled).To(BeFalse(), "Should not have reconciled during error handling")
+			Expect(reconciled).To(BeTrue(), "Should have reconciled during error handling")
```

Comment:
Have you double checked that this change is wanted ?

Reply:
yes, i did that change manually :) i standardized the remove handling